### ISSUE TITLE
`TextArea` folder in dist/addons should be `Textarea`

### ISSUE THREAD

**Issue by 0b1kn00b** [Semantic-Org/Semantic-UI-React#516]
or ' ./Textarea/Textarea''  in index.js

on ubuntu/linux 

```
Unable to resolve some modules:

"./Textarea/Textarea" in /media/i/test_apollo/meteor-starter-kit/node_modules/stardust/dist/addons/index.js (web.browser)
```

**Comment by levithomason** [comment]
Hm, these folders and files are correctly cased for me.  Which version are you running?

`master`, v0.44.9

**addons/index.js**

``` js
export { default as TextArea } from './TextArea/TextArea'
```

**TextArea path**

```
addons/TextArea/TextArea.js
```

**Comment by 0b1kn00b** [comment]
0.35.1 npm, couldn't build git, so went back to it.

**Comment by levithomason** [comment]
Until we v1, there is no branch support for previous versions.  Please update and try again.  LMK if there are any further issues.

**Comment by levithomason** [comment]
It is worth noting there have been a massive amount of bug fixes and updates since 0.35 that you'll likely want to have :)

https://github.com/TechnologyAdvice/stardust/blob/master/CHANGELOG.md

If you're looking for a more incremental update (not recommended), you could update to `0.36.0`.  [It appears](https://github.com/TechnologyAdvice/stardust/pull/409) that is where the TextArea issue was fixed.
